cmd/tiller-proxy/Dockerfile.dev

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.6
```

Comment:
can we not run the binary directly, or with telepresence?

Reply:
true, I'm just not that used to telepresence :)